woocommerce/woocommerce
# Add back FB extension to Installed marketing extensions list.

### All Submissions:

-   [ x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [ x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [ x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?


### Changes proposed in this Pull Request:

We've received a request (see pbIJXs-2mT-p2#comment-4795) to restore Facebook for Woocommerce to the Installed marketing extensions list. This was previously removed here: #33781

### How to test the changes in this Pull Request:

1. Check out this branch.
2. then navigate the Marketing in your WooCommerce site - [example.site/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing](https://example.site/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing)
3. Confirm that FB is listed in the "Installed marketing extensions" section of the marketing page.


Before this change:

![image](https://user-images.githubusercontent.com/4209011/183943460-2efde29c-3008-4763-a759-25776c2b8623.png)

After this change:
![Screenshot 2022-08-11 at 15 03 46](https://user-images.githubusercontent.com/4209011/184198882-1bc2fa1e-48cd-4d5b-ab16-ef3a4dddd134.jpg)



### Other information:

-   [ x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ x] Have you successfully run tests with your changes locally?
-   [ x] Have you created a changelog file for each project being changed, ie `pnpm changelog add --filter=<project>`?



### FOR PR REVIEWER ONLY:
